### PR TITLE
add pycodestyle to ctest, and update all python code to be conformant

### PR DESCRIPTION
**Pull request by RussTedrake**

### CODE REVIEW

File: src/double_pendulum_as_single_pendulum.py [hunk 1]
```diff
@@ -78,7 +87,7 @@ def _DoCalcVectorOutput(self, context, double_pend_state, unused, torque):
 # Set the initial conditions
 context = simulator.get_mutable_context()
 state = context.get_mutable_continuous_state_vector()
-state.SetFromVector((1.,0.,0.2,0.0))  # (theta1, theta2, theta1dot, theta2dot)
+state.SetFromVector((1., 0., 0.2, 0.))  # (θ₁, θ₂, θ̇₁, θ̇₂)
```
Comment (peteflorence) [inline src/double_pendulum_as_single_pendulum.py]: :) fancy rendering!

**Comment by peteflorence** [comment]
Looks nice! LGTM

**Comment by peteflorence** [comment]
Question -- by what mechanism is the linter automatically called?

Is it because it's a test in CMakeLists, and those are are all run by CI?

**Comment by RussTedrake** [comment]
the linter is called as a test in ctest (only checks compliance, does not change the code).  it's your responsibility to change the code to make it compliant.

**Pull request merged**